citrination_client/client.py

```diff
@@ -67,6 +67,32 @@ def upload_file(self, file_path, data_set_id):
         else:
             return None
 
+    def show_latest_dataset_files(self, dataset_id):
```

Comment:
For all of these, can you add documentation in the format of the the upload_file method. This library is user-facing so it's especially important that it be documented.